app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -1,23 +1,33 @@
 ---
 title: "Get an international relocation payment"
+subcategory: If you're already a teacher
 description: |-
-  Non-UK teachers and trainees in physics and languages could be eligible for financial help in the form of an international relocation payment from the UK government.
+  Non-UK teachers in physics and languages could be eligible for financial help in the form of an international relocation payment from the UK government.
 related_content:
     Train to teach in England as a non-UK citizen : "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
     Teach in England if you trained outside the UK: "/non-uk-teachers/teach-in-england-if-you-trained-overseas"
     Apply for your visa to teach in England: "/non-uk-teachers/visas-for-non-uk-teachers"
     Apply for your visa to train to teach in England: "/non-uk-teachers/visas-for-non-uk-trainees"
```

Comment:
@laurahermionetennant92 just spotted in the related content it links  line 7 and line 10 are about training to teach so these can be deleted as the focus of the page is teachers.

Reply:
Hmmm I actually don't agree as there's a decent chance trainee teachers will also land on this page, particularly those who think they are still eligible?